src/Context/notes-context.js

```diff
@@ -57,8 +73,11 @@ const NoteProvider = ({ children }) => {
     priority: "",
   });
 
+  useEffect(() => {
+    getData();
+  }, [noteState]);
```

Comment:
its better to getData at the time of Login, not everytime the noteState is changed.

Reply:
User will logged in once, and I want to get data when state is updated as the color , pin status can be changed

Reply:
alright.. but you're getting an array of notes in response too.